tutorials/tts/Vits_Training.ipynb

```diff
@@ -0,0 +1,430 @@
+    "&& cd ..\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Upv_LxBIsC51"
+   },
+   "source": [
+    "Let's take a look at the vits.py file\n",
+    "\n",
+    "```python\n",
+    "import pytorch_lightning as pl\n",
+    "\n",
+    "from nemo.collections.tts.models.vits import VitsModel\n",
+    "from nemo.core.config import hydra_runner\n",
+    "from nemo.utils.exp_manager import exp_manager\n",
+    "\n",
+    "# hydra_runner is a thin NeMo wrapper around Hydra\n",
+    "# It looks for a config named vits.yaml inside the conf folder\n",
+    "# Hydra parses the yaml and returns it as a Omegaconf DictConfig\n",
+    "@hydra_runner(config_path=\"conf\", config_name=\"vits\")\n",
+    "def main(cfg):\n",
+    "    # Define the Lightning trainer\n",
+    "    trainer = pl.Trainer(replace_sampler_ddp=False, **cfg.trainer)\n",
+    "    # exp_manager is a NeMo construct that helps with logging and checkpointing\n",
+    "    exp_manager(trainer, cfg.get(\"exp_manager\", None))\n",
+    "    # Define the VITS model, this will construct the model\n",
+    "    model = VitsModel(cfg=cfg.model, trainer=trainer)\n",
+    "    # Let's add a few more callbacks\n",
+    "    trainer.callbacks.extend([pl.callbacks.LearningRateMonitor()])\n",
+    "    # Call lightning trainer's fit() to train the model\n",
+    "    trainer.fit(model)\n",
+    "\n",
+    "\n",
+    "if __name__ == '__main__':\n",
+    "    main()  # noqa pylint: disable=no-value-for-parameter\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6nM-fZO-s75u"
+   },
+   "source": [
+    "Let's take a look at the yaml config\n",
+    "\n",
+    "```yaml\n",
+    "name: &name VITS\n",
+    "\n",
+    "train_dataset: ???\n",
+    "validation_datasets: ???\n",
+    "sup_data_path: null\n",
+    "sup_data_types: null\n",
+    "\n",
+    "phoneme_dict_path: \"scripts/tts_dataset_files/cmudict-0.7b_nv22.10\"\n",
+    "heteronyms_path: \"scripts/tts_dataset_files/heteronyms-052722\"\n",
+    "whitelist_path: \"nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv\"\n",
+    "```\n",
+    "\n",
+    "The first part of the yaml defines dataset parameters used by VITS. Then in the head of 'model' section there are processing - related parameters. You can see\n",
+    "that the sample rate is set to 22050 for LJSpeech. \n",
+    "\n",
+    "Looking at the yaml, there is `train_dataset: ???` and `validation_datasets: ???`. The ??? indicates to hydra that these values must be passed via the command line or the script will fail.\n",
+    "\n",
+    "Looking further down the yaml, we get to the pytorch lightning trainer parameters.\n",
+    "\n",
+    "```yaml\n",
+    "trainer:\n",
+    "  num_nodes: 1\n",
+    "  devices: 2\n",
+    "  accelerator: gpu\n",
+    "  strategy: ddp\n",
+    "  precision: 32\n",
+    "  # amp_backend: 'apex'\n",
+    "  # amp_level: 'O2'\n",
```

Comment:
do you want to make a further explanation on alternative options for `amp_backend` and `amp_level`? If not, please remove these two comments.